Assets/MixedRealityToolkit-SDK/Features/UX/Prefabs/Buttons.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
@killerantz has been looking into a new button architecture. we should try to use his new work if possible. otherwise, we are going to be replacing these as soon as we open the branch for beta checkins

Reply:
Wither way, we should bring in only what this scene needs.